makefiles/boards/stm32.inc.mk

```diff
@@ -62,5 +62,6 @@ ifeq (stm32flash,$(PROGRAMMER))
 	DEBUGGER =
 	FLASHFILE ?= $(BINFILE)
 	PROG_BAUD ?= 57600
-	FFLAGS = -b $(PROG_BAUD) -w $(FLASHFILE) -g 0x0 $(PORT)
+	BIN_ADDR ?= $(shell echo  $$(($(ROM_START_ADDR) + $(ROM_OFFSET))))
```

Comment:
`ROM_OFFSET` is not set by default, so you'll have to add

```make
ROM_OFFSET ?= 0
```

otherwise you'll end up with

```
/bin/sh: 1: arithmetic expression: expecting primary: "0x08000000 + "
```